### PR TITLE
add int8 gemm multiply multiply a8w8

### PR DESCRIPTION
**Pull request by valarLip**
add int8 gemm a8w8 fused dequant

### CODE REVIEW

File: profiler/fp8_gmm_profiler.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+EXE="$(find . -name ckProfiler -type f | head -n 1)"
```
Comment (aska-0096) [inline profiler/fp8_gmm_profiler.sh]: please remove this file

File: profiler/int8_gmm_profiler.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
+EXE="$(find . -name ckProfiler -type f | head -n 1)"
```
Comment (aska-0096) [inline profiler/int8_gmm_profiler.sh]: please remove this file

**Review by aska-0096 — CHANGES_REQUESTED** [review]
Please pull the develop branch.

**Review by spolifroni-amd — DISMISSED** [review]
Nothing changed that affects docs.

**Review by aska-0096 — DISMISSED** [review]
LGTM. We can merge it once CI pass.

**Review by carlushuang — APPROVED** [review]
LGTM

**Pull request merged**